src/mlagility/api/script_api.py

```diff
@@ -146,8 +148,27 @@ def benchmark_script(
     models_found: Dict[str, ModelInfo] = {}
 
     for script in input_scripts:
+        script_path, targets, encoded_input = decode_input_script(script)
+
+        # Filter scripts by label
+        if labels != []:
+            skip_script = False
+            user_labels = lab.to_dict(labels)
+            file_labels = lab.load_from_file(encoded_input)
```

Comment:
The naming here gives me a false equivalency between `user_labels` and `file_labels`. Yes, they are both `labels`, but they are not both used to label things (ie, the user put some labels, and the file put some labels). IMO `user_labels` is really more like `required_labels`.

Also suggest renaming `file_labels` to `script_labels`, since we refer to it as `script` everywhere else.

Reply:
Changed names according to suggestion